### PR TITLE
refactoring anchors

### PR DESCRIPTION
**Pull request by oozoofrog**
- add union function to Anchors
- extension functions of Anchors now return Anchors instead of Constraint, and not using AnchorsBuilder
- add create size(width and height) procedure to Anchors+Convenience
- add test case

### CODE REVIEW

File: Sources/SwiftLayout/Extension/Anchors+Convenience.swift [hunk 1]
```diff
@@ -123,9 +129,35 @@ extension Anchors {
...
+    }
+    
+    /// ``Anchors`` for width, height toward toItem: ``ConstraintableItem``
+    ///
+    /// - Parameters:
+    ///  - toItem: constraint second item, ``ConstraintableItem``
+    ///  - offset: constant
+    ///
+    /// - Returns: ``Constraint``
+    public static func size<I: ConstraintableItem>(_ toItem: I, offset: CGFloat = .zero) -> Anchors {
```
Comment (gmlwhdtjd) [inline Sources/SwiftLayout/Extension/Anchors+Convenience.swift:145]: How about use CGSize?

**Review by gmlwhdtjd — APPROVED** [review]

File: Sources/SwiftLayout/Extension/Anchors+Convenience.swift (hunk 1 above)
Comment (oozoofrog) [reply Sources/SwiftLayout/Extension/Anchors+Convenience.swift:145]: smart!

**Review by gmlwhdtjd — APPROVED** [review]

File: Sources/SwiftLayout/Extension/Anchors+Convenience.swift [hunk 2]
```diff
@@ -123,9 +129,56 @@ extension Anchors {
...
+    
+    /// ``Anchors`` for width, height toward toItem: ``ConstraintableItem``
+    ///
+    ///
+    /// - Parameters:
+    ///  - toItem: constraint second item, ``ConstraintableItem``
+    ///  - length: constant
+    ///
+    /// - Returns: ``Constraint``
+    public static func size<I: ConstraintableItem>(_ toItem: I, length: CGFloat = .zero) -> Anchors {
```
Comment (gmlwhdtjd) [inline Sources/SwiftLayout/Extension/Anchors+Convenience.swift:146]: I think, "offset" will be better than "length" in this method

File: Sources/SwiftLayout/Extension/Anchors+Convenience.swift [hunk 3]
```diff
@@ -123,9 +129,56 @@ extension Anchors {
...
+        size(toItem, size: .init(width: length, height: length))
+    }
+    
+    /// ``Anchors`` for width, height toward self
+    ///
+    /// - Parameters:
+    ///  - length: constant
+    ///
+    /// - Returns: ``Constraint``
+    public static func size(length: CGFloat = .zero) -> Anchors {
```
Comment (gmlwhdtjd) [inline Sources/SwiftLayout/Extension/Anchors+Convenience.swift:156]: I think, "length" is good here

File: Sources/SwiftLayout/Extension/Anchors+Convenience.swift [hunk 4]
```diff
@@ -123,9 +129,56 @@ extension Anchors {
...
+    }
+    
+    /// ``Anchors`` for CGSize toward toItem: ``ConstraintableItem``
+    ///
+    /// - Parameters:
+    ///  - toItem: constraint second item, ``ConstraintableItem``
+    ///  - size: constants
+    ///
+    /// - Returns: ``Constraint``
+    public static func size<I: ConstraintableItem>(_ toItem: I, size: CGSize = .zero) -> Anchors {
```
Comment (gmlwhdtjd) [inline Sources/SwiftLayout/Extension/Anchors+Convenience.swift:167]: I think, "offset" will be better than "size" in this method

File: Sources/SwiftLayout/Extension/Anchors+Convenience.swift (hunk 2 above)
Comment (oozoofrog) [reply Sources/SwiftLayout/Extension/Anchors+Convenience.swift:146]: oh you right, offset is more fit in this situation.

File: Sources/SwiftLayout/Extension/Anchors+Convenience.swift (hunk 3 above)
Comment (oozoofrog) [reply Sources/SwiftLayout/Extension/Anchors+Convenience.swift:156]: very agreed.

File: Sources/SwiftLayout/Extension/Anchors+Convenience.swift (hunk 4 above)
Comment (oozoofrog) [reply Sources/SwiftLayout/Extension/Anchors+Convenience.swift:167]: this right too.

**Review by gmlwhdtjd — APPROVED** [review]

**Pull request merged**